src/frequenz/sdk/timeseries/logical_meter/_logical_meter.py

```diff
@@ -32,6 +37,7 @@ def __init__(
         self,
         channel_registry: ChannelRegistry,
         resampler_subscription_sender: Sender[ComponentMetricRequest],
+        component_graph: ComponentGraph,
```

Comment:
Shouldn't this be taken from the `microgrid.get()` singleton so it can pick up updates to the component graph? In any case it is fine for me to leave it like this for now, but I'm asking more to think about how we want things to work in the future, to make sure we all point at the same direction.

In any case, using `microgrid.get().component_graph` will not also help if the component graph is updated, because then the formula should also be regenerated. I'm starting to think that we need an actor or something that can notify (to whoever is listening) about component graph changes.

Reply:
We can also subscribe to config changes from the component graph.  I guess we'll need a sprint to update all parts of the sdk once cloud sync starts to support the sdk,  and we start using the config manager.